connector/connect/server/src/main/scala/org/apache/spark/sql/connect/service/SparkConnectServer.scala

```diff
@@ -36,21 +32,21 @@ object SparkConnectServer extends Logging {
     try {
       try {
         SparkConnectService.start(session.sparkContext)
-        SparkConnectService.server.getListenSockets.asScala.foreach { sa =>
-          val isa = sa.asInstanceOf[InetSocketAddress]
-          logInfo(
-            log"Spark Connect server started at: " +
-              log"${MDC(HOST, isa.getAddress.getHostAddress)}:${MDC(PORT, isa.getPort)}")
-        }
+        val isa = SparkConnectService.bindingAddress
```

Comment:
High level, I think this change is fine, I'm mostly wondering if there is a case where we would have multiple listen sockets, e.g. listening on multiple devices?

Reply:
> High level, I think this change is fine, I'm mostly wondering if there is a case where we would have multiple listen sockets, e.g. listening on multiple devices?

@grundprinzip Yes, if we want to listen on all of the network interfaces in a machine to accept connections from any network, then we can add multiple addresses to let the Server listen on.

For SparkConnectService, the NettyServer is built with single address, so the getListenSockets it should only have one item